packages/esm-patient-vitals-app/src/vitals/paginated-vitals.scss

```diff
@@ -13,12 +19,22 @@
   }
 }
 
-.criticallyLow, .criticallyHigh {
+.criticallyHigh span, .criticallyLow span {
```

Comment:
And the same in similar places...

```suggestion
.criticallyHigh > span, .criticallyLow > span {
```

Reply:
@ibacher I have updated the PR to meet this. Thanks!